doctr/models/builder.py

```diff
@@ -48,7 +48,10 @@ def _sort_boxes(boxes: np.ndarray) -> np.ndarray:
             boxes: bounding boxes of shape (N, 4) or (N, 4, 2) (in case of rotated bbox)
 
         Returns:
-            indices of ordered boxes of shape (N,)
+            tuple: indices of ordered boxes of shape (N,), boxes
+            If straight boxes are passed tpo the function, boxes are unchanged
+            else: boxes returned are straight boxes fitted to the straightened rotated boxes
+            so that we fit the lines afterwards to the straigthened page
```

Comment:
for the docstring to be properly rendered, the next lines have to be extra-indented